docs/reference/cluster_manifest.md

```diff
@@ -175,6 +175,9 @@ explanation of `ttl` and `loop_wait` parameters.
 * **slots**
   permanent replication slots that Patroni preserves after failover by re-creating them on the new primary immediately after doing a promote. Slots could be reconfigured with the help of `patronictl edit-config`. It is the responsibility of a user to avoid clashes in names between replication slots automatically created by Patroni for cluster members and permanent replication slots. Optional.
 
+* **standby_cluster**
```

Comment:
can you please add the example manifest for the standby cluster ?

Reply:
added section to a manifests/complete-postgres-manifest.yaml file

Reply:
I think, it also makes sense to mention the required fields to make it work and add a link to the Patroni [docs](https://github.com/zalando/patroni/blob/ee678f61d792c4b9bdd07ba30193e40343cac8d2/docs/replica_bootstrap.rst#standby-cluster)